app/controllers/sign_up/completions_controller.rb

```diff
@@ -81,7 +81,8 @@ def track_completion_event(last_page)
     end
 
     def pii
-      JSON.parse(user_session.fetch('decrypted_pii', '{}')).symbolize_keys
+      pii_string = Pii::Cacher.new(current_user, user_session).fetch_string
+      JSON.parse(pii_string || '{}').symbolize_keys
```

Comment:
```suggestion
      JSON.parse(pii_string || '{}', symbolize_names: true)
```
Fewer ActiveSupport monkeypatches while we're here? Might also save an intermediate hash from being created/GC'd